config/manager/manager.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         args:
         - --enable-leader-election
         image: contour-operator
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Should this be left as `Always` in a dev build scenario where you'd push the same image tag over and over testing?

Reply:
In contour we leave the "source" YAML files as `Always` (for dev), then update it to `IfNotPresent` for the rendered file, which is the one that gets served for the quickstart link. It'd be nice to do the same here, I think.